test/e2e/serial/tests/configuration.go

```diff
@@ -381,7 +382,102 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				}
 				return true, nil
 			}).WithTimeout(5*time.Minute).WithPolling(10*time.Second).Should(BeTrue(), "failed to update RTE container with LogLevel %q", operatorv1.Trace)
+		})
+
+		It("[replaces-47674][images][tier2] should be able to modify the configurable values under the NUMAResourcesOperator CR", Label("images", "tier2"), func(ctx context.Context) {
+			var nroOperObj nropv1.NUMAResourcesOperator
+			nroKey := objects.NROObjectKey()
+			Expect(fxt.Client.Get(ctx, nroKey, &nroOperObj)).To(Succeed())
+			initialNroOperObj := nroOperObj.DeepCopy()
+
+			testMCP := objects.TestMCP()
+			By(fmt.Sprintf("creating new MCP: %q", testMCP.Name))
+			// we must have this label in order to match other machine configs that are necessary for proper functionality
+			testMCP.Labels = map[string]string{"machineconfiguration.openshift.io/role": roleMCPTest}
+			testMCP.Spec.MachineConfigSelector = &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"machineconfiguration.openshift.io/role": depnodes.RoleWorker,
+					"test-id":                                "47674",
+				},
+			}
+			testMCP.Spec.NodeSelector = &metav1.LabelSelector{
+				MatchLabels: map[string]string{getLabelRoleMCPTest(): ""},
+			}
+
+			Expect(fxt.Client.Create(context.TODO(), testMCP)).To(Succeed())
+			defer func() {
+				By(fmt.Sprintf("CLEANUP: deleting mcp: %q", testMCP.Name))
+				Expect(fxt.Client.Delete(context.TODO(), testMCP)).To(Succeed())
+
+				err := wait.With(fxt.Client).
+					Interval(configuration.MachineConfigPoolUpdateInterval).
+					Timeout(configuration.MachineConfigPoolUpdateTimeout).
+					ForMachineConfigPoolDeleted(context.TODO(), testMCP)
+				Expect(err).ToNot(HaveOccurred())
+			}()
+			// keep the mcp with zero machine count intentionally to avoid reboots
+
+			testNG := nropv1.NodeGroup{
+				MachineConfigPoolSelector: &metav1.LabelSelector{
```

Comment:
Can we use `poolName` instead? since this is the new API we would like to adopt and it's common for both OCP and HCP

Reply:
this is reinnovation for an old test that is designed to run on all versions, if we use poolName we need to label this with `ngpoolname` to run it only on 4.18 and avoid it failing on <4.18. Using poolName will cause to skip running the test on < 4.18 because we use one version of tests image

Reply:
ok, so we want to use the MCP to make sure this test can run against older version of NROP. I'm fine with that.

Reply:
Yea, probably no other choice